selftests/functional/test_nrunner.py

```diff
@@ -144,6 +146,18 @@ def test_recipe_exec_3(self):
         self.assertIn("'status': 'finished'", final_status)
         self.assertEqual(res.exit_status, 0)
 
+    @skipUnlessPathExists('/bin/false')
+    def test_custom_exit_codes(self):
+        status_server = "127.0.0.1:%u" % find_free_port()
+        config = {'run.references': ['/bin/false'],
+                  'run.test_runner': 'nrunner',
+                  'runner.exectest.exitcodes.skip': [1],
+                  'nrunner.status_server_listen': status_server,
+                  'nrunner.status_server_uri': status_server,
+                  'run.keep_tmp': True}
+        with Job.from_config(job_config=config) as job:
+            self.assertEqual(job.run(), 0)
```

Comment:
This test does not seem to be in the right place. It is into TaskRun class, but all tests there are related to the command-line `task-run*` subcommand.

Reply:
Yes, makes sense. I'm moving to a specific class.

Reply:
I think it is fine for now. Later we can move it to https://github.com/avocado-framework/avocado/blob/master/selftests/check.py as it is testing the Avocado features.